app/containers/Settings/index.js

```diff
@@ -77,6 +82,8 @@ class Settings extends Component {
       summary,
       profilePic
     } = fields || '';
+    const WEBVIEW_REF = 'webview';
+    const feedback = 'https://devsummit.io/feedback.html'
```

Comment:
can you set the feedback_url in the global constants instead?

Reply:
ok